INN/umbrella-caribbean
# Remove bullets from gallery block list items

## Changes

This pull request makes the following changes:

- Removes the custom `::before` styled list item bullets on the gallery block list items

## Why



For #49

## Testing/Questions

Features that this PR affects:

- Gallery block list items


Questions that need to be answered before merging:

- [ ] Are there any other cases where a list could be added to a post where we wouldn't want the bullets?

Steps to test this PR:

1. Add a gallery block to a post
2. View it on the frontend and verify no list item bullets are present
